sample.env

```diff
@@ -1,9 +1,60 @@
+# Whether or not to disable the landing page
+DISABLE_LANDING_PAGE=false
+
+# The base url where this instance is accessible,
+# including the scheme (eg. http:// or https://),
+# the domain name, and optionally a port.
 NEXT_PUBLIC_BASE_URL=http://localhost:3000
+
 DATABASE_URL=postgres://your-database/db
-SECRET_PASSWORD=minimum-32-characters
+
 SUPPORT_EMAIL=foo@yourdomain.com
+
+##############################
+# Hardening
+
+# A random 32-character secret key used to encrypt user sessions
+SECRET_PASSWORD=minimum-32-characters
+
+# Set to true to require authentication for creating new polls and accessing admin pages
+AUTH_REQUIRED=false
+
+# Comma separated list of email addresses that are allowed to
+# register and login. Wildcard characters are supported.
+#ALLOWED_EMAILS=*@yourcompany.com
+
+##############################
+# Email providers
+
+# default (lower priority)
 SMTP_HOST=your-smtp-server
 SMTP_PORT=587
 SMTP_SECURE=false
 SMTP_USER=your-smtp-user
 SMTP_PWD=your-smtp-password
+
+## Custom email provider (higher priority, uncomment one to activate)
+# Amazon ses
+#EMAIL_PROVIDER=ses
+#AWS_REGION=
+#API_KEY=
```

Comment:
Some call it api key and others call it access key. There's an opportunity here to be less ambiguous by defining separate env vars for each of these providers.